src/commands/help.js

```diff
@@ -0,0 +1,35 @@
+const Discord = require('discord.js')
+module.exports = {
+  prefix: '!help',
+  /**
+   * @name help
+   * Displays currently available commands.
+   *
+   * @param {Discord.Message} message the message provided
+   */
+
+  /**
+   *Use this format to add new commands:
+   *{name: "Command Name", value: "Command Description"},
+   */
+
+  command: function help(message) {
+    console.log('help', message);
+      const helpEmbed = new Discord.MessageEmbed()
```

Comment:
Wrap a `try{} catch(){}` around the sending of the message so the bot doesn't crash.
in the `catch` be sure to log the error.

Reply:
Added a catch feature